cmd/zc_processor.go

```diff
@@ -90,10 +91,34 @@ func (s *copyTransferProcessor) scheduleCopyTransfer(storedObject storedObject)
 				common.PanicIfErr(err)
 				return string(jsonOutput)
 			} else {
-				// formatting string for remove
-				return fmt.Sprintf("DRYRUN: remove %v/%v",
-					s.copyJobTemplate.SourceRoot.Value,
-					srcRelativePath)
+				// if remove then To() will equal to common.ELocation.Unknown()
+				if s.copyJobTemplate.FromTo.To() == common.ELocation.Unknown() { //remove
```

Comment:
I don't have full context. Can you tell me -
1. Why "Unknown" and not "Trash" too? 
2. Are we supporting dry-run for sync and remove only (and not copy)?

Reply:
@mohsha-msft  There is currently no "Trash" location. If a file is removed the FromTo.To() will read as unknown since the removed file will not have a destination location.
We are supporting dryrun for all copy, remove and sync. The logistic for copy are in copyEnumeratorInit.go